### ISSUE TITLE
Activity toggle isn't keyboard accessible.

### ISSUE THREAD

**Issue by pavkout** [getinsomnia/insomnia#4927]
### Expected Behavior

When I open a document I should be able to navigate between Design, Debug and Test via keyboard.

### Actual Behavior

Currently, this is not supported.

### Is there an existing issue for this?

- [X] I have searched the [issue tracker](https://www.github.com/Kong/insomnia/issues) for this problem.

### Insomnia Version

2022.5.0-beta.2

### What operating system are you using?

macOS

### Operating System Version

macOS Monterey

### Installation method

download from insomnia.rest

**Comment by filfreire** [comment]
Hi @pavkout thanks for reporting this! 

Marking this as an improvement/feature request. We are missing that functionality and seems that it would be really useful. We'll update you here if there's anything released on this.

Feel free report other issues/improvements you come across 👍
